.github/workflows/documentation.yml

```
@@ -0,0 +1,46 @@
+name: Documentation 
+
+on:
+  push:
+    branches: [ main, master, Extensions-Collector-Tool]
+  pull_request:
+    branches: [ main, master, Extensions-Collector-Tool]
+
+defaults:
+  run:
+    working-directory: Tools/Collector
+
+jobs:
+  generateDocumenation:
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      
+    - uses: maxim-lobanov/setup-xcode@v1
+      with:
+        xcode-version: latest-stable
+
+    - name: Build Collector
+      run: |
+        if [ ! -f "collector" ]; then rm collector; fi
+        swift build -c release
+        ln -s .build/release/collector
+
+    - name: Generate Script Commands Documentation
+      run: |
+        ./collector generate-documentation ../../
+
+    - name: Commit files
+      run: |
+        git config --local user.email "bot@raycast.com"
+        git config --local user.name "Raycast Bot"
+        git add ../../raycast-extensions.md ../../raycast-extensions.json
```

How about dropping Raycast in the filename? Doesn't feel that it adds much value. I'm not sure if the JSON is necessary in the repository. I would probably leave it out or do you have any strong reason here?

```suggestion
        git add ../../extensions.md ../../extensions.json
```